### ISSUE TITLE
Can you provide the BERT features of the candidate answers?

### ISSUE THREAD

**Issue by zhangxi1997** [doc-doc/NExT-QA#4]
Hi, thanks for your sharing. 
I wonder can you provide the pre-trained BERT features of the candidate answers?
Thanks a lot!

**Comment by doc-doc** [comment]
Thanks for your interest. As we finetune BERT for multi-choice QA, each answer is appended to the question as a global sentence. If you want the separated candidate feature, you can split the provided global-sentence feature according to the format ```(1): [CLS] question [SEP] option_0 [SEP]```, and the token representations behind the first '[SEP]' belong to the candidate answer.  You need to use BERT's buildin tokenizer to get the tokenized represetantion of the sentence ```(1)``` and find the index of ['SEP'].

**Comment by zhangxi1997** [comment]
Thanks for your quick reply. And I wonder can you provide the finetune BERT to help me extract the candidate answer features?

**Comment by doc-doc** [comment]
Hi, currently, we are not going to release this part of code, but you can refer to this [file](https://github.com/LuoweiZhou/pytorch-pretrained-BERT/blob/master/examples/run_swag.py) for details.

**Comment by junwenchen** [comment]
Hi, 

Thanks for the explanation. Can you let me know what loss is used for fine-tuning bert?

**Comment by doc-doc** [comment]
It's softmax cross-entropy.

**Comment by doc-doc** [comment]
Hi, please find the edited code for fintuning BERT on NExT-QA [here](https://drive.google.com/file/d/1Z0RMnIJrqQcFQEhhuhiRvDvyRmaKdPbg/view?usp=sharing). You can also fine-tune other datasets by using the code.

**Comment by junwenchen** [comment]
Thanks a lot
